src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND_RANDOM_FIGURE_NUMBER = 5;
+    private static final int BOUND_RANDOM_SIDE = 10;
+    private static final double DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_NAME = "DEFAULT FIGURE";
+    private Random random = new Random();
+
+    private int getRandomFigureNumber() {
+        return random.nextInt(BOUND_RANDOM_FIGURE_NUMBER);
+    }
+
+    private double getRandomSide() {
+        return random.nextInt(BOUND_RANDOM_SIDE) + 1;
+    }
+
+    public Figure getRandomFigure() {
+        int randomNumber = getRandomFigureNumber();
+        ColorSupplier randomColor = new ColorSupplier();
```

Comment:
Please make ColorSupplier class variable same as done for Random to avoid creation of new object each time methods are called